code/extensions.py

```diff
@@ -0,0 +1,43 @@
+from .user_settings import get_list_from_csv
+from talon import Module, Context, actions, app
+
+mod = Module()
+mod.list("file_extension", desc="A file extension, such as .py")
+
+_file_extensions_defaults = {
+    "dot pie": ".py",
+    "dot talon": ".talon",
+    "dot mark down": ".md",
+    "dot shell": ".sh",
+    "dot vim": ".vim",
+    "dot see": ".c",
+    "dot see sharp": ".cs",
+    "dot com": ".com",
+    "dot net": ".net",
+    "dot org": ".org",
+    "dot us": ".us",
+    "dot U S": ".us",
```

Comment:
Arguably, `.com`, `.net`, & `.us` are not file extensions (they are TLDs). Is this worth worrying about?

(`.org` is a file extension, because of org-mode in emacs)

Reply:
Is there a use case for splitting these  TLDs into their own list?